src/main.py

```diff
@@ -80,8 +79,8 @@ def execute(node_id, op: Op):
 def main(args):
     nodes = basecases.bootstrap_random_dag(args.num_types, args.complexity, args.nodes_per_type)
 
-    SU = operations.Op.SendUpdateDep
-    demo_ops = {n.node_id: [SU] for n in nodes}
+    SU, BD, RC = operations.Op.SendUpdateDep, operations.Op.Kill, operations.Op.Recover
+    demo_ops = {n.node_id: [SU, BD, RC] for n in nodes}
```

Comment:
in addition to the new ops thread running the random Kill sends, does line 82 add more crash signals in the network?

Reply:
Remove initial ops of kill and recover.